src/store/selectors/index.js

```diff
@@ -0,0 +1,42 @@
+'use babel'
+
+import { createSelector } from 'reselect'
+import ruleDocURIs from 'eslint-rule-documentation'
+import r from '../../f-utils/mini-ramda'
+import { deepProp } from '../../f-utils'
```

Comment:
🔥 Only used in `selectDeepProp`, which isn't used.

Reply:
Yeah, it's not used now. It was just something tiny I threw together while I was in the right mindset, so when the need arose, which is inevitable if we use this, we would already have it ready.